src/js/clients.js

```diff
@@ -724,7 +708,7 @@ if (document.getElementById("featured-clients")) {
 
 if (document.getElementById("all-clients")) {
   const allClientsElement = document.getElementById("all-clients");
-  const allClients = [...featuredClients, ...otherClients];
+  const allClients = otherClients;
```

Comment:
I think this would remove all featured clients

Reply:
Yes, but I don't really understand why we want to display firstly featured clients and then the same clients in the our clients section. So all of the featured clients are displayed twice.